htdocs/societe/card.php

```diff
@@ -1261,6 +1266,16 @@ function init_check_no_email(input) {
                      });';
 				print '</script>'."\n";
 			}
+			if (!empty($conf->multicurrency->enabled)) {
```

Comment:
Using js here has a default. 
If we have filled manually some other field, and have not finished, the form will be submitted before we have finished to fill it.
I think we can completely remove the js part and have the currency modification visible only after the first submit.